BUILD.md

```diff
@@ -0,0 +1,506 @@
+# Conduit Development Guide
+
+:balloon: Welcome to the Conduit development guide! :wave:
+
+This document will help you build, run, and test Conduit from source.
+
+# Table of contents
+
+- [Repo Layout](#repo-layout)
+  - [Control Plane (Go)](#control-plane-go)
```

Comment:
Can you change this link text to "Control Plane (Go/React)" to match the heading? And I think the anchor should be "#control-plane-goreact".